### PR TITLE
fix: check `active` field when verifying access token

### PR DESCRIPTION
**Pull request by tiendq**
in `canPass.verify(token)`

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -30,6 +30,8 @@ export interface VerifyOutput {
   updatedAt: string;
 }
 
+active: true;
```
Comment (manh-vv) [inline src/index.ts:33]: Isn't it expected?

File: src/index.ts [hunk 2]
```diff
@@ -43,14 +45,16 @@ function config(configs: Configs) {
...
   const _accessToken = accessToken.split(' ')[1] || accessToken;
-
-  return _app
+  const result = await _app
     .fetch(`${_app.canPassApi}/authenticate?access_token=${_accessToken}`)
     .then(checkFetchStatus)
     .then(res => res.json());
+
+  // @ts-ignore
+  return result.active ? (result as VerifyOutput) : undefined;
```
Comment (manh-vv) [inline src/index.ts:57]: Here, I would like to update:
* Keep using pure promise style.
* Throw an error if the access_token is not active.

```javascript
function verify(accessToken: string): Promise<VerifyOutput> {
  // handle case access-token start with `Bearer <access-token>`
  const _accessToken = accessToken.split(' ')[1] || accessToken;

  return _app
    .fetch(`${_app.canPassApi}/authenticate?access_token=${_accessToken}`)
    .then(checkFetchStatus)
    .then(res => res.json())
    .then(res => {
      if (!res.active) {
        throw new Error('Access token is not active.');
      }
      return res;
    });
}
```

**Review by manh-vv — CHANGES_REQUESTED** [review]
Hi @tiendq, I reviewed with a request change. Please check.

File: src/index.ts (hunk 1 above)
Comment (tiendq) [reply src/index.ts]: Copy/paste mistake :(. Thanks.

**Review by manh-vv — APPROVED** [review]

**Pull request merged**